cdap-ui/app/hydrator/hydrator-modal.less

```diff
@@ -93,7 +93,10 @@
           max-width: none;
         }
       }
-
+      .validate-btn {
```

Comment:
I noticed that for narrower screen widths, the validate button gets pushed up off the modal and the expand/close buttons also get pushed off screen. 

![Screen Shot 2019-07-12 at 2 09 45 PM](https://user-images.githubusercontent.com/16228294/61159427-e08c4800-a4b0-11e9-8ff6-ceae6474ed9d.png)

Reply:
@yukiej This is a known issue, Ajai mentioned that our modals are not responsive. @ajainarayanan Should I create a JIRA for this or is this already on our radar?

Reply:
@yukiej At what resolution did this break for you? If its anything less than 800px we don't have to worry as of now. We don't support entirely browser sizes that small.

Reply:
@ajainarayanan It depends on the modal title's width. For a file node's modal, it doesn't break until 580px whereas for Joiner, it breaks at 1240px. May be we should reduce max-width of the title? It's currently at 80% width of the modal.

Reply:
@itsanudeep I think we should change the dimenstions we set for the plugin description then. I think we factor in the maximize and close buttons while determining the width of the plugin name and description. May be we should recalculate so that it never breaks down to next line.

Reply:
@ajainarayanan @yukiej Made changes to make sure the buttons won't overflow in modal header.